apps/sms/views/inbox/js/inbox.js

```diff
@@ -251,6 +251,11 @@ var InboxView = {
       titleContainer.replaceChild(groupTitle, title);
     }
 
+    if (!title.textContent) {
+      // display something before waiting for findContact's result
+      title.textContent = threadNumbers[0];
+    }
+
```

Comment:
In the far future I hope we'll be returning conversation with already resolved contact names :)